creator/Documents/DifferencesBetweenBedrockAndJava.md

```diff
@@ -8,95 +8,133 @@ description: "An article covering the major differences between Minecraft: Bedro
 
-## Redstone and Command Differences
+## Command Differences
 
-The structure and implementation of commands between the two versions have diverged as well. Bedrock Edition’s command structure is similar to the system used in versions of Java Edition prior to 1.13. It also forgoes raw JSON strings inside commands for a component-based system. Instead of using long complex JSON strings to customize entities, you can summon an entity with an event to fire, and also name it in a single command.
+The set of commands between the two versions has diverged as well. Java Edition [overhauled many commands](https://minecraft.fandom.com/wiki/Java_Edition_1.13#Command_format_2) in 1.13, while Bedrock made no such changes.
+
+### NBT
+Bedrock notably lacks the ability to specify NBT in any command. This is used in Java Edition to create special items and blocks, summon mobs with specific properties, check for complex conditions, and much more. In some cases, workarounds to achieve similar effects are possible.
+
+For example, compare the syntax of `/summon`:
 
 ```
-summon <entityType: EntityType> [spawnPos: x y z] [spawnEvent: String] [named: String]
+Java:
+/summon <entity> [<position>] [<data>]
+
+Bedrock:
+/summon <entity> [<position>] [<event>] [<name>]
 ```
 
-Currently, there’s no way to `/give` players custom items in Bedrock Edition as you can in Java Edition. The item will need to be created beforehand and teleported to the player. The most common ways of doing this is either by placing the item in a chest and breaking the chest, or making an entity drop it on death via loot table.
+Bedrock can provide an [event to run](../Reference/Content/EntityReference/Examples/EventList.md) on the spawned mob, and a name to be applied.
+
+For giving special items, Bedrock can specify exactly four properties:
+* `can_place_on`
+* `can_destroy`
+* `item_lock`
+* `keep_on_death`
+
+Other properties such as durability, enchantments, custom names, book text, color, and more cannot be specified. As a workaround, [loot tables](../Documents/IntroductionToLootTables.md) can be used in tandem with the `/loot` command in some cases.
+
+### `/execute`
+Bedrock Edition is still using the pre-1.13 `/execute` command, with the following overloads:
+```
+/execute <entity> <position> <command>
+/execute <entity> <position> detect <position> <block> <damage value> <command>
+```
+Java's new `/execute` command, with individual subcommands like `as`, `at`, `if`, `unless`, `positioned`, `store`, `facing`, `align`, and so on is not supported.
 
-Aside from that, commands should feel very familiar between Bedrock Edition and Java Edition versions prior to 1.13. The execute format introduced in Java Edition 1.13 is not supported in Bedrock Edition.
+### Scoreboards
```

Comment:
Elaborate on `/execute` a bit since it's one of the most important commands in the game